test/fixtures/m3u8/fmp4.js

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
This is also new. I denormalized the init segment information down to the segment level, similar to how we handle AES keys right now.